about.html

```diff
@@ -0,0 +1,210 @@
+          <p>The logo of Early 2000's Rock Music Fest 2023 was decided through the logo competition from 8. June to 7. July.</p>
+        </div>
+      </div>
+      <div class="logo-img-container">
+        <div class="logo-img">
+          <img src="./images/logo.svg" alt="logo of the event" width="250px">
+        </div>
+      </div>
+    </div>
+  </section>
+  <hr>
+  <section class="past-events">
+    <div>
+      <div class="title-container">
+        <div class="title">
+          <h2>See the past Early 2000's Rock Music Fests</h2>
+        </div>
+      </div>
+      <div class="separator-container">
+        <div class="separator">
+        </div>
+      </div>
+      <div class="logo-info-container">
+        <div class="logo-info">
+          <p>Take a look at the last two Early 2000's Rock Music Fests witch took place in Los Angeles and Toronto</p>
+        </div>
+      </div>
+        <div class="past-events-box">
+          <div class="past-event-1">
+            <div class="overlay">
+              <div class="past-event">
+                <a href="#"><h3>2019</h3></a>
+              </div>
+              <div class="past-event">
+                <p>Early 2000's Rock Music Fest 2019 in Los Angeles</p>
+              </div>
+            </div>
+          </div>
+          <div class="past-event-2">
+            <div class="overlay">
+              <div class="past-event">
+                <a href="#"><h3>2022</h3></a>
+              </div>
+              <div class="past-event">
+                <p>Early 2000's Rock Music Fest 2022 in Toronto</p>
+              </div>
+            </div>
+          </div>
+        </div>
+    </div>
+  </section>
+  <section class="partners-container">
+    <div class="partner-title">
+      <h2>Partner</h2>
+    </div>
+    <div class="title-underline"></div>
+    <div class="partner-logo-container">
+      <div class="partners">
+          <img src="./images/partner_logos/mozilla.png" alt="">
+        <div>
+          <img src="./images/partner_logos/google.png" alt="">
+        </div>
+        <div>
+          <img src="./images/partner_logos/naver.png" alt="">
+        </div>
+        <div>
+          <img src="./images/partner_logos/daumkakao.png" alt="">
+        </div>
+        <div>
+          <img src="./images/partner_logos/airbnb.png" alt="">
+        </div>
+      </div>
+    </div>
+  </section>
+  <footer>
+    <div class="container">
+      <div class="row row-cols-md-2">
+        <div class="col-6 d-flex justify-content-center cc-logo-container">
+          <img class="positive" src="./images/footer/positive.png" alt="creative commons korea logo in positive">
+          <img class="negative" src="./images/footer/negative.png" alt="creative commons korea logo in negative">
+        </div>
+        <div class="col-6 d-flex justify-content-center cc-logo-container">
+            <p>2023 Creative Commons korea.<br>Some Rights Reserved.</p>
+        </div>
```

Comment:
- [ ] Kindly fix the space between the image and the copyright.

<img width="1359" alt="Screenshot 2023-02-17 at 15 03 25" src="https://user-images.githubusercontent.com/81444338/219814201-0586951a-cac7-4c03-a77f-c95bf452720d.png">

Reply:
Thank you so much for the review @Kossi-stack . I made all the changes mentioned.